examples/OPL2AudioBoard/SerialMidi/SerialMidi/SerialMidi.ino

```diff
@@ -0,0 +1,308 @@
+/**
+ * This example can be used together with a Teensy 2.0 or later to use the OPL2 Audio Board as a MIDI device. To
+ * configure the Teensy as a MIDI device set USB Type to MIDI in the IDE using Tools > USB Type > MIDI. Once connected
+ * the board should appear in the device list as 'OPL2 AUdio Board MIDI'. You can now use test the board with, for
+ * example, MIDI-OX, your favorite music creation software or DosBox!
```

Comment:
Can you please update the comment for the example? Please include the name / reference to the MIDI library that should be installed for the example. I tested it with https://github.com/FortySevenEffects/arduino_midi_library

Also feel free to credit yourself :)

Reply:
Updated comments, made the requested changes. They should be in teh latest commit.

Cheers, and thanks for the great library!